examples/overview.ipynb

```diff
@@ -0,0 +1,806 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "fig, ax = diff.residue_contacts.plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "atom_pairs = list(map(list, [k for k in frame_contacts.atom_contacts.counter.keys()]))"
+   ]
+  },
```

Comment:
Is this leftover debug code or just some code without description (this is quite the invocation)?
Please either remove or add some description
```suggestion
```